.github/workflows/build.yml

```diff
@@ -5,11 +5,8 @@ name: Build
 # 2. It includes the PR/Commit text in the action item
 # 3. Artifacts are not available between workflows.
 
-# This is only allowing pushes on the moonbeam repo for pull requests.
-####### DO NOT CHANGE THIS !! #######
 on:
   pull_request:
-    types: [opened, edited, unassigned, synchronize]
```

Comment:
Not sure why this is removed. I suppose this will now be triggered when there is a comment or other types of (non-code) changes no ?

Reply:
By default, it's only retriggered when the code is modified, and I realize with use that it's better to go back to the default behavior.